### PR TITLE
#253791 [bugfix] Silent booking failure on session expire

### PR DESCRIPTION
**Pull request by chetan-thapliyal**

### CODE REVIEW

File: src/Resources/js/modules/ajax-forms.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+import AjaxErrorHandler from "./ajax-error-handler";
```
Comment (hacksch) [inline src/Resources/js/modules/ajax-forms.js:1]: please us single quotes

File: src/Resources/js/modules/custom-functions.js [hunk 2]
```diff
@@ -1,3 +1,5 @@
+import AjaxErrorHandler from "./ajax-error-handler";
```
Comment (hacksch) [inline src/Resources/js/modules/custom-functions.js:1]: single quotes

File: src/Resources/js/modules/participation-request-handler.ts [hunk 3]
```diff
@@ -1,4 +1,5 @@
 import {ParticipationResponse} from "./participation-response-handler";
```
Comment (hacksch) [inline src/Resources/js/modules/participation-request-handler.ts:1]: single quotes in both lines

File: src/Resources/js/views/adminParticipationEdit.ts [hunk 4]
```diff
@@ -1,5 +1,6 @@
 import {CombinedMealDialog, SerializedFormData} from "../modules/combined-meal-dialog";
```
Comment (hacksch) [inline src/Resources/js/views/adminParticipationEdit.ts:1]: single quotes

File: src/Resources/js/views/mealIndex.ts [hunk 5]
```diff
@@ -8,6 +8,7 @@ import {MealOfferUpdate, MealOfferUpdateHandler} from "../modules/meal-offer-upd
 import {ParticipationUpdateHandler} from "../modules/participation-update-handler";
```
Comment (hacksch) [inline src/Resources/js/views/mealIndex.ts:8]: single quotes

File: src/Resources/js/views/slotIndex.ts [hunk 6]
```diff
@@ -1,4 +1,5 @@
 import Switchery from "switchery.js";
+import AjaxErrorHandler from "../modules/ajax-error-handler";
```
Comment (hacksch) [inline src/Resources/js/views/slotIndex.ts:2]: single quotes

File: src/Resources/js/modules/custom-functions.js (hunk 2 above)
Comment (chetan-thapliyal) [reply src/Resources/js/modules/custom-functions.js]: done

File: src/Resources/js/modules/participation-request-handler.ts (hunk 3 above)
Comment (chetan-thapliyal) [reply src/Resources/js/modules/participation-request-handler.ts]: done

File: src/Resources/js/views/mealIndex.ts (hunk 5 above)
Comment (chetan-thapliyal) [reply src/Resources/js/views/mealIndex.ts]: done

File: src/Resources/js/views/slotIndex.ts (hunk 6 above)
Comment (chetan-thapliyal) [reply src/Resources/js/views/slotIndex.ts]: done

**Pull request merged**